ci/bump

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+if [ -z "$1" ]; then
+    echo "Usage: bump component [version]"
+    exit 1;
+fi
+
+comp=$1; shift;
+
+cd components/${comp}
+cz_bump_out=`cz bump --files-only "$@"`
+commit_title=`echo "${cz_bump_out}" | head -1`
+commit_body=`cat ../../release_notes.txt`
```

Comment:
I have executed `ci/bump mdns`
and it showed `cat: ../../release_notes.txt: No such file or directory`

Reply:
Good point, thanks! The error reporting is really poor!

Updated the script to run commitizen bump command with `--dry-run` and only if it succeeds to continue with the process.